site/pages/docs/ref/add-cal/add-cal-en.hbs

```diff
@@ -6,7 +6,7 @@
 	"categoryfile": "plugins",
 	"description": "Add an event to a calendar (google, Apple, Android, Outlook and others).",
 	"altLangPrefix": "add-cal",
-	"dateModified": "2022-07-25"
+	"dateModified": "2025-07-25"
```

Comment:
This change might've been made in error (2025-07-25 is 2+ months in the future).